app/graphql/resolvers/concerns/record.rb

```diff
@@ -0,0 +1,17 @@
+module Resolvers
+  module Concerns
+    module Record
+      extend ActiveSupport::Concern
+
+      included do
+        type "Types::#{self::MODEL_CLASS}".safe_constantize, null: false
+
+        argument :id, String, 'Global ID for Record', required: true
+      end
+
+      def resolve(id:)
+        RecordLoader.for(self.class::MODEL_CLASS).load_by_global_id(id)
```

Comment:
Nit: I wonder if we need to ensure all these constants in a concern are fully qualified (i.e. `::RecordLoader`). It's caused problems in the past with models, but idk if it's a special case with Rails objects.